assets/queries/cloudFormation/ecs_task_definition_with_plaintext_password/test/positive.yaml

```diff
@@ -0,0 +1,32 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Description: A sample template
+Resources:
+  taskdefinition:
```

Comment:
Change "taskdefinition" to "taskdefinition4". Sorry I missed this earlier.

Reply:
done